superset-frontend/src/chart/Chart.jsx

```diff
@@ -81,21 +84,74 @@ const defaultProps = {
 class Chart extends React.PureComponent {
   constructor(props) {
     super(props);
+    this.state = {
+      textColor: 'black'
+    }
+
     this.handleRenderContainerFailure = this.handleRenderContainerFailure.bind(
       this,
     );
+    this.getExtraData = this.getExtraData.bind(this);
   }
 
   componentDidMount() {
     if (this.props.triggerQuery) {
       this.runQuery();
     }
+    this.getExtraData();
   }
 
   componentDidUpdate() {
     if (this.props.triggerQuery) {
       this.runQuery();
     }
+    this.getExtraData();
+  }
+  
+  getExtraData() {
```

Comment:
Just want to understand these logic... basically if the vizType is BigNumber, it will trigger an extra query to the database the color column? I suppose the query/request is the same as the 'original' one. @lacardenasv

Reply:
Yes, you're right, it's the same plus the color column